docs/contributing/testing.md

```diff
@@ -130,12 +124,12 @@ _When running as CLI, the results can be viewed as printed out to the command sc
 
 ## Cross browser testing
 
-You can read more from [Mozilla](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Introduction) on cross browser testing and processes.
+You can read more from [Mozilla](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Introduction) on cross-browser testing and processes.
 
-Click on the links below to report browser bugs.
+Report browser bugs by following these links.
```

Comment:
nit: colon instead of period here

Reply:
Changed.
I think there should be colons before code examples as well. That seems to be the pattern used for other MS tech documentation: https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/numbers-in-csharp?tutorial-step=4. Making changes to follow that convention as well.